Modules/xSharePoint/DSCResources/MSFT_xSPTimerJobState/MSFT_xSPTimerJobState.schema.mof

```diff
@@ -0,0 +1,36 @@
+/*
+**Description**
+
+This resource is used to configure a timer job and make sure it is in a specific state.
+The resource can be used to enable or disabled the job and configure the schedule of the job.
+
+The schedule parameter has to be written in the SPSchedule format (https://technet.microsoft.com/en-us/library/ff607916.aspx). Examples are:
```

Comment:
Can we make sure that every sentence in the documentation starts on a new line (it's a preferred git thing for the wiki pages we generate from these )